dev/NavigationView/NavigationView.cpp

```diff
@@ -3324,3 +3501,43 @@ bool NavigationView::IsFullScreenOrTabletMode()
 
     return isFullScreenMode || isTabletMode;
 }
+
+void NavigationView::Expand(winrt::NavigationViewItem const& value)
+{
+
+}
+
+void NavigationView::Collapse(winrt::NavigationViewItem const& value)
+{
+
+}
+
+winrt::TreeViewNode NavigationView::NodeFromContainer(winrt::DependencyObject const& container)
+{
+    if (auto lv = GetActiveListView())
+    {
+        if (auto navListView = lv.try_as<winrt::NavigationViewList>())
+        {
+            return winrt::get_self<NavigationViewList>(navListView)->NodeFromContainer(container);
+        }
+    }
+    return nullptr;
+}
+
+winrt::DependencyObject NavigationView::ContainerFromNode(winrt::TreeViewNode const& node)
+{
+    if (auto lv = GetActiveListView())
+    {
+        if (auto navListView = lv.try_as<winrt::NavigationViewList>())
+        {
```

Comment:
since we always do null check and try_as, move logic to GetActiveListView and make GetActiveListView returns NavigationViewList*?